src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -366,6 +366,14 @@ namespace DataHVACGlobals {
         OptStartDataType() = default;
     };
 
+    // Compressor operation
+    enum class CompressorOperation
+    {
+        Unassigned = -1,
```

Comment:
I think @mitchute is doing a pass that is converting `Unassigned` to `Invalid`.

Reply:
I will change this one in this PR.